setup.py

```diff
@@ -68,7 +68,6 @@ def find_version(*file_paths):
     keywords=["Epoch of Reionization", "Cosmology"],
     install_requires=[
         "click",
-        # 'tqdm',
```

Comment:
We don't use progress bars? tqdm is a pretty good package for that. At any rate if it was commented out I guess we don't need it.

Reply:
No, we don't at the moment! We should, but it's not there yet, so no reason to include it.